documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-verify.adoc
+// getting-started.adoc
+
+[id='setup-external-clients-{context}']
+= Setting up access for clients outside of Kubernetes
+
+This procedure shows how to configure client access to a Kafka cluster from outside Kubernetes.
+
+Using the address of the Kafka cluster, you can provide external access to a client on a different Kubernetes namespace or outside Kubernetes entirely.
+
+You configure an external Kafka listener to provide the access.
+
+The following external listener types are supported:
+
+* `route` to use OpenShift `Route` and the default HAProxy router
```

Comment:
```suggestion
* `route` to use OpenShift `Route` and the default HAProxy router.
```

Reply:
I don't think we need these full stops. The bullets aren't full sentences.